CHANGELOG.md

```diff
@@ -37,9 +39,14 @@
     * Upgrade Jackson to version 2.9.9
 
   New Feature
```

Comment:
This doesn't look right :confused:

Reply:
@rohanKanojia: I reset my branch with master branch and this was in one of the commits.

Reply:
Could you please fix it?